requirements.txt

```diff
@@ -1,8 +0,0 @@
-matplotlib
-scipy
-numpy>=1.19.5
-pandas
-nbsphinx
-networkx
-xlwt
```

Comment:
poetry newbie question : how does pip and users of pip will deal without this requirement file? (cf https://pip.pypa.io/en/stable/user_guide/#requirements-files)

Reply:
pip understands pyproject.toml and will install everything as usual.